docs/Contribution.md

```diff
@@ -13,13 +13,45 @@ The instructions given below are for software developers who want to work on [ou
 
 3. Create a new issue, a community member will get back to you and once approved, send a pull request for the same.
 
-4. Please go through the issue list first (both open as well as closed) amd make sure the issue you are reporting does not replicate the issues already reported. Clearly explain what the exact issue is and provide the steps to recreate it as well, if possible.
+## Avoid doing the following mistakes
+1. Fix a new issue and submit a PR without reporting and getting it approved at first.
 
-5. If you have issues on multiple pages, then report them separately. Do not combine them into a single issue.
+2. Fix an issue assigned to somebody else and submit a PR before the assignee does. 
 
-6. All the PRs are needed to be sent to the [develop](https://github.com/systers/macc) branch. Do not add merge commits to your PR.
```

Comment:
@medhach  Oh sorry . I just meant to remove the line "Do not add merge commits to your PR" as I have moved it into the section "PR Best Practices". I will restore the line  before that.